java/src/test/android/app/src/androidTest/java/ai/onnxruntime/example/javavalidator/SimpleTest.kt

```diff
@@ -92,13 +112,14 @@ class SimpleTest {
                         output.use {
                             @Suppress("UNCHECKED_CAST")
                             val rawOutput = output[0].value as Array<Array<FloatArray>>
+                            val precision = if (useQNN) 1e-3 else 1e-6
```

Comment:
Is 1e-3 the minimum required for the tests to pass?

Reply:
No, I will test the value to find the maximum allowed value. As the QNN team said they would need to check if the threshold really need to be changed or there is some other test issue, I added e-3 to pass the test. The actual value would be between e-3 and e-4

Reply:
QNN EP uses fp16 ops